src/Microsoft.HttpRepl.IntegrationTests/Commands/CommandTestsBase.cs

```diff
@@ -19,11 +19,13 @@ public class CommandTestsBase
         protected void ArrangeInputs(string parseResultSections,
             out MockedShellState shellState,
             out HttpState httpState,
-            out ICoreParseResult parseResult)
+            out ICoreParseResult parseResult,
+            int caretPosition = 0,
+            string responseContent = null)
         {
-            parseResult = CoreParseResultHelper.Create(parseResultSections);
+            parseResult = CoreParseResultHelper.Create(parseResultSections, caretPosition);
             shellState = new MockedShellState();
-            httpState = GetHttpState(string.Empty);
+            httpState = GetHttpState(responseContent);
```

Comment:
It looks like you change the default from `string.Empty` to `null` here. Just want to make sure that doesn't have any downstream implications. Guessing not, but thought I'd check.

Reply:
Nope, it should have any implications. I changed it as it won't allow string.Empty as default. Added null check here just in case:
https://github.com/aspnet/HttpRepl/commit/c696814b69478ae5a0162d45379a90f5a7ea384c

Reply:
string.empty is readonly, but not const (default params must be constant values).  I believe you can use `""` as a constant non-null empty string value for parameter defaults.  Or use continue to use null, and make sure it's handled correctly everywhere.

Reply:
"" works, updated here:
https://github.com/aspnet/HttpRepl/commit/f0d84962d91caabf832097b6957e32b1fce4c265

Thanks!